dwave_networkx/drawing/chimera_layout.py

```diff
@@ -28,33 +28,34 @@
 
 
 def chimera_layout(G, scale=1., center=None, dim=2):
-    """Positions the nodes of graph G in a Chimera cross topology.
```

Comment:
wrt [this comment](https://github.com/dwavesystems/dwave-networkx/pull/226/files/fcbd1a4c551d06d36685233e45ee56cb3298afab#r1134612372), could we delete the phrase "in a Chimera cross topology"?

Reply:
This function creates a Chimera layout that can then be plotted. It could have potentially realized the layout in different ways, in particular, the cross and column layouts shown [here](https://docs.dwavesys.com/docs/latest/c_gs_4.html). The modifier "in a Chimera cross topology" is saying that the function selected to implement the cross layout. 
"topology" is not the best word choice because it's used for the QPU connectivity (i.e. Chimera structure). So the info is needed but needs improved phrasing.

Reply:
"in a Chimera cross layout"?

Reply:
Might be okay or perhaps something like, "Positions a graph's nodes in a Chimera layout with unit cells as crosses."
Kelly uses "cross configuration" in Pegasus but I'm not sure that's clear to most users.